base/docs/basedocs.jl

```diff
@@ -161,7 +161,7 @@ end
     include expressions, literal values, and symbols. For example:
 
         macro sayhello(name)
-            return :( println("Hello, ", \$name) )
+            return :( println("Hello, ", $name) )
```

Comment:
This is inside a docstring, so this `$` needs to be escaped... If you do `help> macro` you'll see it correctly printing without the `\`. Did you see this render incorrectly somewhere?

The other change is good though.